src/web/templates/admin/tournaments/tournament_card.html

```diff
@@ -109,68 +109,89 @@
 {% endblock %}
 
 {% block card_buttons_left %}
-    <span {{ macros.tooltip_attributes('info', _('Edit the properties of the tournament.'), 'top') }}>
-        <button
-            class="btn btn-sm btn-outline-primary text-nowrap"
-            hx-get="{{ url_for('admin-tournament-modal', action='update', event_uniq_id=admin_event.uniq_id, tournament_id=tournament.id) }}"
-            hx-target="body"
-            hx-indicator="#please-wait"
-            preload
-            data-bs-toggle="modal"
-            data-bs-target="#modal-wrapper"
-        >
-            <i class="bi-pencil-fill pe-1"></i>
-            {{ _('Edit') }}
+    <div class="dropup">
+        <button class="btn btn-sm btn-outline-primary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+            {{ _('Actions') }}
         </button>
-    </span>
-
-    {% if EXPERIMENTAL_FEATURES %}
-        <div class="dropdown d-inline">
-            <button class="btn btn-sm btn-outline-primary dropdown-toggle mx-auto" type="button" data-bs-toggle="dropdown" aria-expanded="false">
-                <i class="bi-download pe-1"></i>{{ _('Export') }}
-            </button>
-            <ul class="dropdown-menu locale">
-                {% for exporter in tournament_exporters %}
-                    <li
-                        class="dropdown-item text-start cursor-pointer"
-                        onclick="window.open('{{ url_for('admin-tournament-export', event_uniq_id=admin_event.uniq_id, tournament_id=tournament.id, exporter_id=exporter.id) }}', '_blank')"
-                        {{ macros.tooltip_attributes('info', exporter.tooltip, 'right') }}
+        <ul class="dropdown-menu">
+            <li class="dropdown-submenu">
+                <a class="dropdown-item d-flex justify-content-between" href="#">
```

Comment:
The ``href="#"`` updates the URL, can we remove it?
```suggestion
                <a class="dropdown-item d-flex justify-content-between">
```